foreshadow/steps/mapper.py

```diff
@@ -66,28 +52,8 @@ def transform(self, X, *args, **kwargs):
         Returns:
             result from .transform()
 
-        Raises:
-            ValueError: if not fitted.
-
-        """
-        if self.feature_processor is None:
-            raise ValueError("not fitted.")
-        Xt = self.feature_processor.transform(X, *args, **kwargs)
-        return Xt
-
-    def fit_transform(self, X, *args, **kwargs):
-        """Fit then transform the cleaner step.
-
-        Args:
-            X: the data frame.
-            *args: positional args.
-            **kwargs: key word args.
-
-        Returns:
-            A transformed dataframe.
-
         """
-        return self.fit(X, *args, **kwargs).transform(X)
+        return super().transform(X=X)
```

Comment:
Do we need to override this method then?

Reply:
Same as above.